app/models/component/manifest/app_container.rb

```diff
@@ -37,6 +37,11 @@ def environment_settings
         env << { name: '_JAVA_OPTIONS', value: '-Xmx1280m -Xms1280m' }.with_indifferent_access
         env << { name: '_BUSBAR_BUILD_TIME', value: timestamp.to_s }.with_indifferent_access
         env << { name: 'PORT', value: app_port.to_s }.with_indifferent_access
+        env << { name: 'DATADOG_LOGS_NAME', value: app_id }.with_indifferent_access
+        env << { name: 'BUSBAR_APP', value: app_id }.with_indifferent_access
+        env << { name: 'BUSBAR_ENV', value: environment.name }.with_indifferent_access
+        env << { name: 'BUSBAR_COMPONENT', value: type }.with_indifferent_access
+        env << { name: 'BUSBAR_NODETYPE', value: node.id }.with_indifferent_access
```

Comment:
Neither `kubectl get nodes -o wide` nor `kubectl describe nodes` provides the value for `node.id`. So I'm not sure what I'm looking at here. [Kubernetes » Docs » Nodes](https://kubernetes.io/docs/concepts/architecture/nodes/) doesn't mention node IDs, either. What are some examples of `node.id`s that I would expect to see when running `echo $BUSBAR_NODETYPE` inside of a given container?

Reply:
You can find it on the code: https://github.com/busbar-io/busbar-server/blob/e5a72e91cd0305ce3a66a3c543a686c124e05fac/app/models/component/manifest.rb#L49

You would find the value of the Kubernetes label.

Reply:
Is there a `kubectl` command that I could run to see an example of what those labels look like?

Reply:
Could the values for this be `node`, `ruby`, `java`, etc.?